src/main/java/de/caritas/cob/agencyservice/api/admin/service/agency/AgencyAdminFullResponseDTOBuilder.java

```diff
@@ -50,6 +50,15 @@ private AgencyAdminResponseDTO createAgency() {
         .createDate(String.valueOf(this.agency.getCreateDate()))
         .updateDate(String.valueOf(this.agency.getUpdateDate()))
         .deleteDate(String.valueOf(this.agency.getDeleteDate()));
+
+    if (hasAnyNonNullDemographicsAttribute()) {
```

Comment:
this is a theoretical case, but I have noticed it in Topics as well (and with the test-data)

If the feature is disabled, but there is still data, we output the data.
Here it would be simply the data, for Topics a list with null-values.

But I think for now this is ok.

Reply:
yes, however I think the "topics" solution is a bit worse, because then frontend needs to decide whether there are any topic information or not (they cannot have simple if (topics != null/undefined) check).
I think it's better to return null object if there is no data instead of an empty object.
Since we are not merging it back, we can adapt builder for topic in the open source.

Reply:
yes, I just stumbled because of the tests, because the test-data are there not different for each scenario.